wgpu-hal/src/auxil/dxgi/factory.rs

```diff
@@ -141,7 +141,7 @@ pub fn create_factory(
         }
         // If we don't print it to info as all win7 will hit this case.
         Err(err) => {
-            log::info!("IDXGIFactory1 creation function not found: {:?}", err);
+            log::warn!("IDXGIFactory1 creation function not found: {:?}", err);
```

Comment:
The comment above needs to be updated, since it refers to "info", here and in two other places.